index.js

```diff
@@ -170,8 +179,15 @@ AFRAME.registerSystem('input-mapping', {
   }
 });
 
-AFRAME.registerInputActions = function (inputActions) {
+AFRAME.registerInputActions = function (inputActions, defaultActionSet) {
   AFRAME.inputActions = inputActions;
+  if (typeof defaultActionSet !== 'undefined') {
+    if (AFRAME.inputActions[defaultActionSet]) {
+      AFRAME.currentInputMapping = defaultActionSet;
+    } else {
+      console.error(`input-mapping: trying to set a non registered action set '${defaultActionSet}'`);
```

Comment:
Should `currentInputMappings` become a property with a setter so we can do this check when it is set directly as well?

Reply:
Yep, I'll work on that on another PR as I will create a new scope `AFRAME.INPUT`? as we are already including so many objects on the AFRAME scope.